dashboard/lib/pd/foorm/form_analytics_parser.rb

```diff
@@ -0,0 +1,112 @@
+module Pd::Foorm
+  # Responsible for taking Forms (ie, a survey comprised of many questions)
+  # from our in-house survey system (Foorm)
+  # and processing them into a "one row per question" format
+  # for export to CSV, and, ultimately, to our analytics database (Redshift).
+  class FormAnalyticsParser
+    include Constants
+    extend Helper
+
+    HEADERS = %w(
+      form_id
+      form_name
+      form_version
+      question_type
+      question_name
+      question_text
+      matrix_item_name
+      matrix_item_text
+      is_facilitator_specific
+      response_options
+      num_response_options
+    )
+
+    def self.reshape_all_forms_and_export_to_csv
```

Comment:
For each of these "reshape" methods, it would be helpful to have a header comment that briefly describes the input and output formats.